modules/axhal/src/platform/aarch64_bsta1000b/misc.rs

```diff
@@ -26,7 +27,8 @@ pub fn reset_cpu() {
     reset_qspi();
 
     //Data Width = 32
-    let ptr = phys_to_virt((axconfig::A1000BASE_SAFETYCRM + 0x8).into()).as_mut_ptr() as *mut u32;
+    let ptr = phys_to_virt((axconfig::devices::A1000BASE_SAFETYCRM + 0x8).into()).as_mut_ptr()
```

Comment:
this line is exactly the same with line 12,
can we use a const global variable.